File: C++/Programming Basics with C++/07. Exercise - Nested Conditional Statement/simple_task/NewHouse.cpp
#include <iostream>
#include <string>
#include <cmath>

using namespace std;

int main() {
    string typeFlowers;
    int flowersCount;
    int budget;
    cin >> typeFlowers >> flowersCount >> budget;

    double totalPrice = 0;

    if (typeFlowers == "Roses") {
        totalPrice = flowersCount * 5.0;
        if (flowersCount > 80) {
            totalPrice *= 0.9;
        }
    } else if (typeFlowers == "Dahlias") {
        totalPrice = flowersCount * 3.8;
        if (flowersCount > 90) {
            totalPrice *= 0.85;
        }
    } else if (typeFlowers == "Tulips") {
        totalPrice = flowersCount * 2.8;
        if (flowersCount > 80) {
            totalPrice *= 0.85;
        }
    } else if (typeFlowers == "Narcissus") {
        totalPrice = flowersCount * 3.0;
        if (flowersCount < 120) {
            totalPrice *= 1.15;
        }
    } else if (typeFlowers == "Gladiolus") {
        totalPrice = flowersCount * 2.5;
        if (flowersCount < 80) {
            totalPrice *= 1.20;
        }
    }

    double leftMoney = budget - totalPrice;

    cout.setf(ios::fixed);
    cout.precision(2);

    if (leftMoney >= 0) {
        cout << "Hey, you have a great garden with " << flowersCount << " " << typeFlowers << " and " << leftMoney
             << " leva left." << endl;
    } else {
        cout << "Not enough money, you need " << abs(leftMoney) << " leva more." << endl;
    }

    return 0;
}

File: C#/Programming Basics with C#/07. Nested Conditional Statements - Exercise/05. New House/NewHouse.cs
using System;

class NewHouse
{
    static void Main()
    {
        string flourType = Console.ReadLine();
        int flourNum = int.Parse(Console.ReadLine());
        int budget = int.Parse(Console.ReadLine());
        double totalPrice = 0.0;
        double leftMoney = 0.0;

        if (flourType == "Roses")
        {
            totalPrice = flourNum * 5.0;
            if (flourNum > 80)
            {
                totalPrice = totalPrice - totalPrice * 0.1;
            }
        }
        else if (flourType == "Dahlias")
        {
            totalPrice = flourNum * 3.8;
            if (flourNum > 90)
            {
                totalPrice = totalPrice - totalPrice * 0.15;
            }
        }
        else if (flourType == "Tulips")
        {
            totalPrice = flourNum * 2.8;
            if (flourNum > 80)
            {
                totalPrice = totalPrice - totalPrice * 0.15;
            }
        }
        else if (flourType == "Narcissus")
        {
            totalPrice = flourNum * 3.0;
            if (flourNum < 120)
            {
                totalPrice = totalPrice + totalPrice * 0.15;
            }
        }
        else if (flourType == "Gladiolus")
        {
            totalPrice = flourNum * 2.5;
            if (flourNum < 80)
            {
                totalPrice = totalPrice + totalPrice * 0.20;
            }
        }

        leftMoney = budget - totalPrice;

        if (leftMoney >= 0)
        {
            Console.WriteLine($"Hey, you have a great garden with {flourNum} {flourType} and {leftMoney:F2} leva left.");
        }
        else if (leftMoney < 0)
        {
            Console.WriteLine($"Not enough money, you need {Math.Abs(leftMoney):F2} leva more.");
        }
    }
}